### DISCUSSION TITLE
Can I stop the miner?

### DISCUSSION THREAD

**Discussion by intellq** [RainbowMiner/RainbowMiner#2192] [Q&A] [answered]
I'd like an option to stop the mining if earnings is below a threshold.

Example 01: if earning < 0.25 usd / day = stop mining and wait
Example 02: if earning < 0.0000125 btc / day = stop mining and wait

I'm not talking about profit, like after deducing the energy. My energy cost is 0. I know I can artificially set a high energy cost to try stop the mining.

But I just want if plain earnings is less than this threshold, then stop mining in the rig.

Is it possible?

**Comment by RainbowMiner** [comment, marked as answer]
Not yet, but it sounds like a smart enhancement to me. I'll add a minimum earning level parameter.
